frontend/src/bundles/home/components/HeaderLandingNav.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <div>
+    <v-app-bar
+      flat
+      height="prominent"
+      class="menu-header"
+    >
+      <v-container>
+        <v-row
+          align="center"
+        >
+          <v-toolbar-title>
+            <v-img
+              class="mr-md-16"
+              src="@/assets/image/main_logo.svg"
+            />
+          </v-toolbar-title>
+          <v-toolbar-items class="hidden-md-and-down">
+            <v-btn
+              v-for="(item, index) in items"
+              :key="index"
+              depressed
+              :to="item.url"
+              :ripple="false"
+              class="menu-header--nav text-capitalize font-weight-bold mr-3"
+            >
+              {{ item.title }}
+            </v-btn>
+          </v-toolbar-items>
+
+          <v-spacer />
+          <v-btn
+            v-if="!isAuthorized"
+            class="hidden-md-and-down text-capitalize font-weight-bold mr-3"
+            color="button-main"
+            large
+            raised
+            rounded
+            plain
+            depressed
+            max-width="162"
+            width="100%"
+            :ripple="false"
+            to="/auth/sign-in"
```

Comment:
Please use route name for navigation instead of a path

Reply:
fixed